libs/components/layout/testing/src/modules/description-list/description-list-description-harness.ts

```diff
@@ -0,0 +1,27 @@
+import { ComponentHarness } from '@angular/cdk/testing';
+
+/**
+ * Harness for interacting with a description list description component in tests.
+ * @internal
```

Comment:
Should be public since it's accessible via the public `with` method of the harness.